analysis/README.md

```diff
@@ -33,9 +33,9 @@ machine Main with degree: 16 {
     reg Y[<=];
     reg A;
 
-    instr identity X -> Y = sub.identity;
-    instr one -> Y = sub.one;
-    instr nothing = sub.nothing;
+    instr identity X -> Y = sub::identity;
```

Comment:
Same here: `sub` is a value, so `sub.identity` is a member access and not a path access.

Reply:
Oh, I see, I thought we stopped using `.` in general. I'll fix it. Thanks!